src/main/java/com/minecolonies/rankup/util/Constants.java

```diff
@@ -0,0 +1,59 @@
+package com.minecolonies.rankup.util;
+
+/**
+ * Constants for our plugin.
+ */
+public final class Constants
+{
+
+    private Constants()
+    {
+        //Empty private constructor.
+    }
+
+    public static class PlayerInfo
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this class "final" or add a public constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS2974)